src/components/structures/MatrixChat.js

```diff
@@ -946,6 +953,7 @@ module.exports = React.createClass({
                 this.state.screenAfterLogin.screen,
                 this.state.screenAfterLogin.params,
             );
+            // XXX: is this necessary? `showScreen` should do it for us.
```

Comment:
That's what I thought, but I didn't really want to change it at the same time :)